doc/QUICK_START.md

```diff
@@ -92,14 +92,21 @@ These are generic (and condensed) installation instructions for the **current de
 
         foreman start -f Procfile.dev
 
+   The application will be available at `http://127.0.0.1:8080`. To customize
```

Comment:
@NobodysNightmare good catch! Probably because there are actually two `Procfile`, one for production and one for dev (this was the best solution @kgalli and I could come up with)